### PR TITLE
Clean up unused setter and simplify getter

### PR DESCRIPTION
**Pull request by micgro42**
That setter became unused in #130 / 003c8e7

### CODE REVIEW

**Review by lucaswerkmeister — CHANGES_REQUESTED** [review]
eslint points out the setter import is now unused (but I can’t add a comment on it, let alone a suggested change, because github)

**Review by lucaswerkmeister — APPROVED** [review]
Is there a reason why the `store.dispatch( HANDLE_LANGUAGE_CHANGE, newLanguage )` can’t be in the computed setter, so it’s still more consistent with the other fields? :thinking:

**Comment by lucaswerkmeister** [comment]
> Is there a reason why the `store.dispatch( HANDLE_LANGUAGE_CHANGE, newLanguage )` can’t be in the computed setter, so it’s still more consistent with the other fields? thinking

Hm, that doesn’t seem to work, but I don’t know why…

**Comment by micgro42** [comment]
> > Is there a reason why the `store.dispatch( HANDLE_LANGUAGE_CHANGE, newLanguage )` can’t be in the computed setter, so it’s still more consistent with the other fields? thinking
> 
> Hm, that doesn’t seem to work, but I don’t know why…

It seems to work for me and I like it more. I updated the PR :)

File: src/components/NewLexemeForm.vue [hunk 1]
```diff
@@ -41,8 +40,11 @@ const language = computed( {
 	get(): SearchedItemOption | null {
 		return store.state.language;
 	},
-	set( newLanguage: SearchedItemOption | null ): void {
-		store.commit( SET_LANGUAGE, newLanguage );
+	async set( newLanguage: SearchedItemOption | null ): Promise<void> {
+		await store.dispatch( HANDLE_LANGUAGE_CHANGE, newLanguage );
+		if ( newLanguage ) {
+			store.commit( CLEAR_PER_FIELD_ERRORS, 'languageErrors' );
```
Comment (lucaswerkmeister) [inline src/components/NewLexemeForm.vue:46]: Hm, should this really wait until we’re done looking up the language code of the language item? (But can be switched around later.)

**Review by lucaswerkmeister — APPROVED** [review]
I think I just copied the wrong code or something ^^

File: src/components/NewLexemeForm.vue (hunk 1 above)
Comment (micgro42) [reply src/components/NewLexemeForm.vue:46]: Yeah, let's think about this later

**Pull request merged**